client/types.go

```diff
@@ -65,17 +68,48 @@ const (
 func (l ConnectConsistencyLevel) String() string {
 	switch l {
 	case ConnectConsistencyLevelAny:
-		return "ConnectConsistencyLevelAny"
+		return "any"
```

Comment:
Out of curiosity, is there any reason the consistency levels are iota's instead of strings? If they were strings you wouldn't have to do all this manual switching. Is it just a perf thing?

Reply:
Yeah it's a perf thing, don't want to have to encode strings over the wire when making RPCs.  We may not even push these over the wire just yet, but it was a "best practices" thing considering we weren't sure if we were going to use the nodes as coordinators or not at that point.